src/nft/pages/collection/index.tsx

```diff
@@ -168,13 +155,7 @@ const Collection = () => {
                 </Box>
 
                 {/* @ts-ignore: https://github.com/microsoft/TypeScript/issues/34933 */}
-                <AnimatedBox
-                  position={isMobile && isFiltersExpanded ? 'fixed' : 'static'}
-                  style={{
-                    transform: gridX.to((x) => `translate(${x as number}px)`),
-                    width: gridWidthOffset.to((x) => `calc(100% - ${x as number}px)`),
-                  }}
-                >
+                <Box flex="1" position={isMobile && isFiltersExpanded ? 'fixed' : 'static'}>
```

Comment:
Box is an older pattern we are working on removing from the library.  it's a bit confusing

If you are adding or replacing code use styled-components.  So I would just use a styled.div here

Reply:
Let's follow up on Slack on this one. Would definitely like to learn a bit more about the migration strategy and pros/cons switching to Styled components instead of primitives like this one

Reply:
@aballerr I think I'd prefer to stick to whatever is currently used by a given file until we come up with a general plan for styling (post-launch, most likely). My main concern is introducing a lot of styled components that can effectively reduce the readability of the file. We do have some interesting action items in this context, so let's continue on Slack.